### ISSUE TITLE
How did you get the tower boundary files in boundaries directory?

### ISSUE THREAD

**Issue by su-hyeonjeong** [genforce/idinvert_pytorch#38]
Thank you for your great work!

How did you get the tower boundary files in boundaries directory?
using interfaceGAN?

Plus, how can I get the other class (own dataset, not face, tower, bedroom) boundary files ???

**Comment by zhujiapeng** [comment]
Yes, the provided boundaries are obtained using InterFaceGAN. For other classes, you also can use InterFaceGAN or other methods such as the one in [here](https://github.com/zhujiapeng/resefa).
